hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieInputFormatUtils.java

```diff
@@ -324,14 +326,24 @@ public static HoodieTimeline getHoodieTimelineForIncrementalQuery(JobContext job
    * Extract HoodieTableMetaClient from a partition path (not base path)
    */
   public static HoodieTableMetaClient getTableMetaClientForBasePathUnchecked(Configuration conf, Path partitionPath) throws IOException {
```

Comment:
can we add a specific UT for this method alone? to cover different logic branches in the method